bcda/service/service_test.go

```diff
@@ -170,7 +173,7 @@ func (s *ServiceTestSuite) TearDownTest() {
 	conf.SetEnv(s.T(), "PRIORITY_ACO_REG_EX", s.priorityACOsEnvVar)
 }
 
-func (s *ServiceTestSuite) TestIncludeSuppressedBeneficiaries() {
+func (s *ServiceTestSuite) TestIncludeSuppressedBeneficiariesIntegration() {
```

Comment:
I really liked the strategy outlined in the second response in this [post](https://stackoverflow.com/questions/25965584/separating-unit-tests-and-integration-tests-in-go) about how to discern between unit and integration tests when running tests, hence the name updates.

Reply:
Quick thought: What about `TestIncludeSuppressedBeneficiaries_Integration`?  Makes it pop out more to me.

Is there a case to be made for adding the `t.Short` clause or do we pretty much always want to run everything?

Reply:
yes and yes! I like the snake_case to make it more visible and eventually, we would want to include the `if t.short; t.skip()` line in the integration tests!